### ISSUE TITLE
Spheres on Internet explorer have lower resolution

### ISSUE THREAD

**Issue by dk5ras** [JeremyHeleine/WP-Photo-Sphere#16]
Hi,

don't know for sure if it may be some setting, but at least on "stock" MS internet explorers the plugin brings low pic quality. Especially along edges of objects raw steps become visible, while a Chrome on the same machine brings the same sphere in perfect quality. This applies for both preview and full window modes. Tested this on Win 7, 8 and 10, with identical results.

Otherwise a great plugin, brought the first real use case for my 360 deg cam :)

With best regards

Ralph.

**Comment by JeremyHeleine** [comment]
Hi,

Are we talking about Internet Explorer or Edge? Is it possible to have the version number?

**Comment by dk5ras** [comment]
Yes, one affected Internet Explorer I have right now at hand is 11.0.9600.18793 on Win 8, and Edge 40.15063.0.0 on Win 10 does the same. On both machines Chrome shows the original quality.

**Comment by dk5ras** [comment]
Here some affected 360 deg views - not that something with my installation is strange:

http://dk5ras.dyndns.org/www/db0fue/20170520/
